src/components/Facepile/Jquery.Facepile.js

```diff
@@ -0,0 +1,200 @@
+        $personaCard.removeClass('is-active').hide();
+      });
+
+      $panel.on("click", ".js-togglePanel", function() {
+        $panel.toggleClass("is-open");
+      });
+
+      /** Open oveflow panel with list of members */
+      $facePile.on("click", ".js-overflowPanel", function(event) {
+        $panelMain.css({display: "block"});
+        $panel.toggleClass("is-open")
+              .removeClass('ms-FacePile-panel--addPerson')
+              .addClass('ms-FacePile-panel--overflow ms-Panel--right');
+      });
+
+      /** Display person count on page load */
+      $(document).ready(function() {
+        $(".ms-FacePile-overflowText").text("+" + $membersCount);
+      });
+
+      /** Show selected members from PeoplePicker in the FacePile */
+      $('.ms-PeoplePicker-result').on('click', function() {
+        var name = $(this).find(".ms-Persona-primaryText").html();
+        var title = $(this).find(".ms-Persona-secondaryText").html();
+        var selectedInitials = (function() {
+          var nameArray = name.split(' ');
+          var nameInitials = '';
+          for (i = 0; i < nameArray.length; i++) {
+            nameInitials += nameArray[i].charAt(0);
+          }
+
+          return nameInitials.substring(0,2);
+        })();
+        var selectedClasses = $(this).find('.ms-Persona-initials').attr('class');
+        var selectedImage = $(this).find('.ms-Persona-image').attr('src');
+
+        var facePileItem = 
+          '<button class="ms-FacePile-itemBtn ms-FacePile-itemBtn--member" title="' + name + '">' +
+            '<div class="ms-Persona ms-Persona--xs">' +
+              '<div class="ms-Persona-imageArea">' +
+                '<div class="' + selectedClasses + '">' + selectedInitials + '</div>' +
+                '<img class="ms-Persona-image" src="' + selectedImage + '">' +
+              '</div>' +
+              '<div class="ms-Persona-presence"></div>' +
+              '<div class="ms-Persona-details">' +
+                '<div class="ms-Persona-primaryText">' + name + '</div>' +
+                '<div class="ms-Persona-secondaryText">' + title + '</div>' +
+              '</div>' +
+            '</div>' +
+          '</button>';
+
+        /** Add new item to members list in facepile */
+        $membersList.prepend(facePileItem);
+
+        /** Increment member count */
+        incrementMembers();
+      });
+
+      /** Remove members in panel people picker */
+      $pickerMembers.on('click', '.js-selectedRemove', function() {
+        var memberText = $(this).parent().find('.ms-Persona-primaryText').text();
+
+        var $facePileMember = $membersList.find(".ms-Persona-primaryText:contains(" + memberText + ")").first();
+
+        if ($facePileMember) {
+          $facePileMember.parent().closest('.ms-FacePile-itemBtn').remove();
+
+          $membersCount -= 1;
+
+           /** Display a maxiumum of 5 people */
+           $(".ms-FacePile-members").children(":lt(5)").show();
```

Comment:
We're using a lot of hard-coded 5s.  Would it make sense to put this into a variable?